lib/org/chromium/webidl/Outputer.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'Outputer',
+
+  documentation: 'Outputer for parsed Web IDL',
+
+  properties: [
+    {
+      class: 'String',
+      documentation: 'Buffer for current output run.',
+      name: 'buf_',
```

Comment:
It's a convention that declares this property isn't meant for external use. It's like 'protected', but without actual language protection (given that JS doesn't have such a thing).